assets/app/view/game/spreadsheet.rb

```diff
@@ -65,6 +69,25 @@ def render_history_titles(corporations)
         or_history(corporations).map { |turn, round| h(:th, @game.or_description_short(turn, round)) }
       end
 
+      def render_player_history
+        # OR history should exist in all
+        zebra_row = true
+        last_values = nil
+        @game.players.first.history.map do |h|
```

Comment:
cleaner to do map.with_index 

index.even? 
then manual zebra rowing

Reply:
it's not exactly the same, as the values aren't shown if they are identical